### ISSUE TITLE
Alpha channel hidden, alpha channel still visible in HEX Color Input

### ISSUE THREAD

**Issue by harmenvm** [kleinfreund/vue-accessible-color-picker#18]
## Steps to reproduce
Hi, 

I've disabled the alpha channel, but the alpha channel is still visible inside the hex color. 
I'm aware of the previous bugfix #17 

The code: 
```
  <ColorPicker
    :visible-formats="['hex', 'hsl', 'rgb']"
    color="#23a96a"
    alpha-channel="hide"
    default-format="hex"
  ></ColorPicker>
  ```

## Current result
On load the color will be cut off 
#23a96a will show as #23a9

After moving the picker 
#23a9 will show as #25a468ff, with the alpha channel init.  
(The color is an example, depends what color you pick ofcourse)

Side note: 

Here you see the colorChange function for example: 
const colorChange = (eventData) => {
 console.log(eventData.cssColor);
 console.log(eventData.colors.hex);
 }

On load this function fires and logs the following: 
#23a6
#23a69a

So on load the color.hex is correct. 

After picking a color it logs:
#658784
#658784ff 

now the css color is correct, but the colors.hex ain't. 

## Expected result
The color should not be cut off, nor display the alpha channel.

## Environment

- **vue-accessible-color-picker** version: ^4.0.2 

I'm using the newest version 4.0.2, 

I might think it's a bug, or what am I doing wrong? 

Thank you in advance :)

**Comment by kleinfreund** [comment]
Hey there. Thank you for the report. It turned out I didn’t sufficiently test the functionality with color values that don’t have an alpha channel.

I’ve implemented a fix for this in https://github.com/kleinfreund/vue-accessible-color-picker/commit/9f31c3eb39d4960b66fb097d083cc8c79c4b3e12. It was released in package version [4.0.3](https://github.com/kleinfreund/vue-accessible-color-picker/releases/tag/v4.0.3).

**Comment by harmenvm** [comment]
Thank you!
